plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-order-summary-block/test/block.js

```diff
@@ -46,6 +46,13 @@ const defaultUseStoreCartValue = {
 	cartHasCalculatedShipping: mockPreviewCart.has_calculated_shipping,
 };
 
+jest.mock( '@wordpress/data', () => {
+	return {
+		__esModule: true,
+		...jest.requireActual( 'wordpress-data-wp-6-7' ),
```

Comment:
Once we update wp-data to 6.6, then I think we can remove this hack. 🙌  

It doesn't need to be 6.7 but just needs a version that supports lock-unlock. Since it's only used in the test environment and for only these specific tests, using 6.6 or 6.7 does not affect the results.